files/conf/wait-for-domains

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-domains="{{.Cluster.Etcd.Domain}} {{.Cluster.Kubernetes.API.Domain}} quay.io"
+domains="{{.Cluster.Etcd.Domain}} {{.Cluster.Kubernetes.API.Domain}}"
```

Comment:
Why were we waiting for quay?

Reply:
dunno

Reply:
this was quite important as it fixed flaky errors that were caused by AWS DNS that was at startup being able to resolve internal AWS DNS names but not public DNS so waiting until it could resolve something from outside was helping  work around such buggy state

This will probabbly cause issues again,  so I dont really agree here with the removal.

Reply:
It would be good to add it back with a comment and maybe a different domain (because we probably gonna abandon quay).